com.bestaticpy.bestatic.yml

```diff
@@ -0,0 +1,12 @@
+id: com.bestaticpy.bestatic
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: bestatic
+modules:
+  - python3-Bestatic.json
+finish-args:
```

Comment:
finish args is always found above modules

Reply:
Done.